packages/table/src/headers/headerCell.tsx

```diff
@@ -73,6 +77,7 @@ export interface IHeaderCellProps extends IProps {
 export interface IInternalHeaderCellProps extends IHeaderCellProps {
     /**
      * Specifies if the cell is reorderable.
+     * @deprecated since 1.21.0; pass `isReorderable` to `ColumnHeader` or `RowHeader` instead
```

Comment:
Geoff what do you think about this? I moved the logic for adding the `TABLE_HEADER_REORDERABLE` class from here into `Header`, so this prop serves no purpose here anymore. See: https://github.com/palantir/blueprint/pull/1250/files#diff-34e8a9161a519e81e1d774377ef4c28cR323